qucs-core/build-aux/ar-lib

```diff
@@ -0,0 +1,270 @@
+#! /bin/sh
```

Comment:
I think this file is generated by the autotools and so it should not be committed.
We should likely add it to `.gitignore`